exposing-to-internet/deploy.sh

```diff
@@ -0,0 +1,146 @@
+
+echo "Installing apigeecli"
+curl -s https://raw.githubusercontent.com/apigee/apigeecli/master/downloadLatest.sh | bash
+export PATH=$PATH:$HOME/.apigeecli/bin
+
+TOKEN="$(gcloud auth print-access-token)"
+
+# Get Org and instance information
+ORG_JSON=$(apigeecli organizations get -o "$PROJECT" -t "$TOKEN")
+INSTANCE_JSON=$(apigeecli instances list -o "$PROJECT" -t "$TOKEN")
+NETWORK=$(echo "$ORG_JSON" | jq --raw-output .authorizedNetwork)
+INSTANCE_NAME=$(echo "$INSTANCE_JSON" | jq --raw-output '.instances[0].name')
+REGION=$(echo "$INSTANCE_JSON" | jq --raw-output '.instances[0].location')
+SERVICE_ATTACHMENT=$(echo "$INSTANCE_JSON" | jq --raw-output '.instances[0].serviceAttachment')
+ENVIRONMENT_NAME="sample-environment"
+ENVIRONMENT_GROUP_NAME="sample-environment-group"
+
+# Create and attach a sample Apigee environment
+echo -n "Creating environment..."
+OPERATION=$(apigeecli environments create -o "$PROJECT" -e "$ENVIRONMENT_NAME" -d PROXY -p PROGRAMMABLE -t "$TOKEN" | jq --raw-output '.name' | awk -F/ '{print $4}')
+wait_for_operation "$OPERATION"
+
+echo -n "Attaching environment to instance (may take a few minutes)..."
+OPERATION=$(apigeecli instances attachments attach -o "$PROJECT" -e "$ENVIRONMENT_NAME" -n "$INSTANCE_NAME" -t "$TOKEN" | jq --raw-output '.name' | awk -F/ '{print $4}')
+wait_for_operation "$OPERATION"
+
+# Enable APIs
+gcloud services enable compute.googleapis.com --project="$PROJECT" --quiet
+
+# Reserve an IP address for the Load Balancer"
+echo "Reserving load balancer IP address..."
+gcloud compute addresses create sample-apigee-vip --ip-version=IPV4 --global --project "$PROJECT" --quiet
+RUNTIME_IP=$(gcloud compute addresses describe sample-apigee-vip --format="get(address)" --global --project "$PROJECT" --quiet)
+RUNTIME_HOST_ALIAS="$ENVIRONMENT_GROUP_NAME".$(echo "$RUNTIME_IP" | tr '.' '-').nip.io
+
+# Create a sample Apigee environment group and attach the environment
+echo -n "Creating environment group..."
+OPERATION=$(apigeecli envgroups create -o "$PROJECT" -d "$RUNTIME_HOST_ALIAS" -n "$ENVIRONMENT_GROUP_NAME" -t "$TOKEN" | jq --raw-output '.name' | awk -F/ '{print $4}')
+wait_for_operation "$OPERATION"
+
+echo -n "Attaching environment to group..."
+OPERATION=$(apigeecli envgroups attach -o "$PROJECT" -e "$ENVIRONMENT_NAME" -n "$ENVIRONMENT_GROUP_NAME" -t "$TOKEN" | jq --raw-output '.name' | awk -F/ '{print $4}')
+wait_for_operation "$OPERATION"
+
+# Create a Google managed SSL certificate
+echo "Creating SSL certificate..."
+gcloud compute ssl-certificates create sample-apigee-ssl-cert \
+    --domains="$RUNTIME_HOST_ALIAS" --project "$PROJECT" --quiet
+
+## Create a global Load Balancer
+echo "Creating external load balancer..."
+
+# Create a PSC NEG
+gcloud compute network-endpoint-groups create sample-apigee-neg \
+  --network-endpoint-type=private-service-connect \
+  --psc-target-service="$SERVICE_ATTACHMENT" \
+  --region="$REGION" \
+  --network="$NETWORK" \
+  --subnet="${SUBNET:-default}" \
```

Comment:
cant expect to have "default" to be available 
So instead of relying on this, lets include this as part of the env.sh and get the values sourced

The script failed for me as my project does not have a default subnet